src/components/TextInputs/NumberInput.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { PureComponent } from 'react'
+import {
+  NativeSyntheticEvent,
+  Platform,
+  StyleSheet,
+  TextInput,
+  TextInputKeyPressEventData,
+  TextInputProps,
+} from 'react-native'
+import { Colors } from '../../res'
+import { ThemeContext } from '../Contexts'
+
+interface Props extends TextInputProps {
+  onChangeText: (text: string) => void
+  digit?: string
+  focus?: boolean
```

Comment:
Are we using this `focus?` prop anywhere?

Reply:
Nope. Removing it.